src/mixins/multiselectApi.js

```diff
@@ -0,0 +1,64 @@
+import { get } from 'lodash';
+
+export default {
+  props: {
+    value: null,
+    placeholder: String,
+    trackBy: {
+      type: String,
+      default: 'id',
+    },
+    label: {
+      type: String,
+      default: 'name',
+    },
+    api: {
+      type: String,
+      default: 'process',
+    },
+    multiple: {
+      type: Boolean,
+      default: false,
+    },
+    storeId: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  data() {
+    return {
+      options: [],
+      selectedOption: null,
+    };
+  },
+  watch: {
+    value: {
+      immediate: true,
+      handler(value) {
+        this.selectedOption = this.storeId
+          ? this.options.find(option => get(option, this.trackBy) == value)
+          : value;
+        value && !this.selectedOption ? this.loadSelected(value) : null;
+      },
+    },
+  },
+  methods: {
+    change(value) {
+      this.$emit('input', this.storeId ? get(value, this.trackBy) : value);
+    },
+    loadOptions(filter) {
+      window.ProcessMaker.apiClient
+        .get(this.api + (typeof filter === 'string' ? '?filter=' + filter : ''))
```

Comment:
`window.ProcessMaker.apiClient` is an instance of [`axios`](https://github.com/axios/axios), so you can speicfy the query string (`?` part) of the URL through an options object:
```suggestion
        .get(this.api, { params: { filter } })
```